rfc/trace_identifiers.md

```diff
@@ -44,10 +44,18 @@ A string is preferred over other formats for the following reasons:
 * Backwards compatibility with pre-existing ID formats.
 * Strongly supported across many languages, and commonly used for transferring data between independent subsystems.
 
-## Alternate Formats
-In some cases, additional formats may be appropriate, if a language supports multiple common transport formats. Rather than manually converting the string value to another format, an additional accessors could be added to allow the tracer to do the conversion.
+## Why `ToTraceID` instead of `TraceID`?
 
-If tracing systems converge on common in-memory formats for Trace-Context identifiers, accessors may be added for those formats as well.
+The method names `ToTraceID` and `ToSpanID` were chosen over `TraceID` and `SpanID` for two reasons:
+
+1) To avoid conflicts with tracer implementations, which often use `TraceID` and `SpanID` to expose their identifiers in their native format.
+2) To communicate to the user that a format conversion may be occuring, and thus there may be some cost to calling these methods.
+
+## Future Formats
```

Comment:
Changing the header from "alternate" to "future" seems to say "we may add other methods to the official spec in the future for these reasons" rather than just "current tracing implementations may have other methods for these reasons". Was that your intention? If so, we could say so more explicitly.